frontend/src/metabase/metadata/components/TableBreadcrumbs/TableBreadcrumbs.tsx

```diff
@@ -0,0 +1,82 @@
+import cx from "classnames";
+
+import {
+  skipToken,
+  useGetTableQuery,
+  useListDatabaseSchemasQuery,
+} from "metabase/api";
+import { Ellipsified } from "metabase/core/components/Ellipsified";
+import { Flex, Group, Icon } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+import S from "./TableBreadcrumbs.module.css";
+
+interface Props {
+  className?: string;
+  hideTableName?: boolean;
```

Comment:
This component will be used in another place where `hideTableName` will be `false` (see [SEM-210](https://linear.app/metabase/issue/SEM-210/show-database-and-table-of-each-segment-in-the-segment-admin-list))